introspection-engine/core/src/error_rendering.rs

```diff
@@ -17,12 +17,20 @@ pub fn render_error(crate_error: Error) -> UserFacingError {
             .unwrap()
             .into()
         }
+        Error::DatamodelError(errors) => {
+            let error = user_facing_errors::UnknownError {
+                message: errors.to_pretty_string("schema.prisma", schema),
+                backtrace: Some(format!("{:?}", user_facing_errors::new_backtrace())),
+            };
+
+            user_facing_errors::Error::from(error)
```

Comment:
Looking at https://github.com/prisma/prisma-engines/blob/83ac782d5d93dcee37efeba8ccbeff596701148a/libs/user-facing-errors/src/lib.rs#L67 it looks like this could maybe be replaced with `Error::new_non_panic_with_current_backtrace(errors.to_pretty_string("schema.prisma", schema))`

Reply:
Adapted it.